set.mm

```diff
@@ -359471,10 +359479,20 @@ already know is total (except at ` 0 ` ).  There are branch points at
       $.
   $}
 
+  $( Lemma for ~ leibpi .  (Contributed by Mario Carneiro, 7-Apr-2015.)  (Proof
+     shortened by Steven Nguyen, 23-Mar-2023.) $)
+  leibpilem1 $p |- ( ( N e. NN0 /\ ( -. N = 0 /\ -. 2 || N ) ) ->
+    ( N e. NN /\ ( ( N - 1 ) / 2 ) e. NN0 ) ) $=
```

Comment:
The proof as well as the usages seem to show that there is not much gain in having these two theorems bundled like that into one. Also, `-. N = 0` is implied. I think two theorems (and not merely lemmas) would be justified:
```
|- ( ( N e. NN0 /\ -. 2 || N ) -> N e. NN ) $=
|- ( ( N e. NN0 /\ -. 2 || N ) -> ( N - 1 ) / 2 ) e. NN0 ) $=
```
and maybe also
```
|- ( ( N e. NN0 /\ -. 2 || N ) -> ( N + 1 ) / 2 ) e. NN ) $=
```